src/cronMatcher.ts

```diff
@@ -170,3 +180,11 @@ export function parseCrontabRanges(matches: string[], source: string) {
   );
   return { minutes, hours, dates, months, dows };
 }
+
+export const parseCronString = (pattern: string, source: string = "") => {
```

Comment:
Please add an explicit return type.

Reply:
Also, please rename to `parseCronRangeString` since it's only responsible for parsing the "range" part of a cron string :+1: